ait/dsn/plugins/raf_plugin.py

```diff
@@ -53,3 +30,18 @@ def __del__(self):
         self.raf_object.stop()
         self.raf_object.unbind()
         self.raf_object.disconnect()
+
+    def _transfer_data_invoc_handler(self, pdu):
+        """"""
+        frame = pdu.getComponent()
+        if "data" in frame and frame["data"].isValue:
+            tm_data = frame["data"].asOctets()
+        else:
+            err = (
+                "RafTransferBuffer received but data cannot be located. "
+                "Skipping further processing of this PDU ..."
+            )
+            ait.core.log.info(err)
```

Comment:
Should this be logged as an error (or is info intended)?

Reply:
This info is present in the existing RAF class– I just copied it over.